.circleci/config.yml

```diff
@@ -6,18 +6,10 @@ jobs:
       # see https://circleci.com/developer/images/image/cimg/openjdk
       - image: cimg/openjdk:8.0.442
     steps:
-      - restore_cache:
-          keys:
-            - v1-dependencies-{{ checksum "${CIRCLE_PROJECT_REPONAME}/pom.xml" }} # appends cache key with a hash of pom.xml file
```

Comment:
missed this in #120 , but in retrospect it's obvious that you can't restore the m2 cache from a prior run if the cache key is the pom.xml which hasn't been checked out yet. I ditched this caching mechanism entirely now since:
1) we don't pull that much to build this pom
2) the storage/isolation of caches is not documented? I found the circleci docs on how to use/clear the cache pretty detailed, but exactly how they isolated the read/write from it is unclear. Not super worried about cache poisoning, although this project does release a single jarfile for the maven site skin.